### PR TITLE
Engdocs 1260

### PR DESCRIPTION
**Pull request by aevesdocker**
PR refreshes Hub content

### CODE REVIEW

File: docker-hub/image-management.md [hunk 1]
```diff
@@ -4,69 +4,34 @@ keywords: dashboard, images, image management, inactive
 title: Advanced Image Management dashboard
 ---
 
-{% include upgrade-cta.html
-  body="Docker Pro, Team, and Business users can now use Docker Hub's **Advanced Image Management** dashboard. The dashboard enables a new level of access to your content, allowing fine-grained pruning of old data and exploration of old versions of pushed images. Upgrade now to start using the Advanced Image Management dashboard."
-  header-text="This feature requires a paid Docker subscription"
-  target-url="https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade_image-mgmt"
-%}
+{% include upgrade-cta.html s%}
```
Comment (crazy-max) [inline docker-hub/image-management.md:7]:
```suggestion
{% include upgrade-cta.html %}
```

File: docker-hub/convert-account.md [hunk 2]
```diff
@@ -4,60 +4,44 @@ title: Convert an account into an organization
...
+3. From the **Convert Account** tab, select **Convert to Organization**.
 
-5. As part of the conversion, you must enter a **Docker ID** to set an organization owner. This is the user account that will manage the organization, and the only way to access the organization settings after conversion. You cannot use the same Docker ID as the account you are trying to convert.
+4. Review the warning displayed about converting a user account. This action cannot be undone and has considerable implications for your assets and the account.
 
-6. Click **Convert** to confirm. The new owner will receive a notification email. Use that owner account to log into your new organization.
+5. Enter a **Docker ID** to set an organization owner. This is the user account that will manage the organization, and the only way to access the organization settings after conversion. You cannot use the same Docker ID as the account you are trying to convert.
 
-    Your Pro user account has now been converted to an organization.
+6. Select **Convert and Purchase** to confirm. The new owner receives a notification email. Use that owner account to log into your new organization.
```
Comment (craig-osterhout) [inline docker-hub/convert-account.md:47]:
```suggestion
6. Select **Convert and Purchase** to confirm. The new owner receives a notification email. Use that owner account to log in to your new organization.
```

File: docker-hub/deactivate-account.md [hunk 3]
```diff
@@ -4,47 +4,65 @@ description: Learn how to deactivate a Docker Hub account or an organization
...
 
-2. If you belong to any organizations, remove your account from all of them.
+Before deactivating your Docker Hub account, ensure that you meet the following requirements:
 
-3. If you are the sole owner of an organization,  either add someone to the **owners** team and then remove yourself from the organization, or deactivate the organization. Similarly, if you are the sole owner of a company, either add someone else as a company owner and then remove yourself, or deactivate the company.
+- You must not be a member of a company or any teams or organizations. You must remove the account from all teams, organizations, or the company.
 
-4. If you have an active subscription, downgrade it to the **Docker Personal** subscription.
+    To do this:
+    1.  Navigate to **Organizations** and then select the organization(s) you need to leave.
```
Comment (craig-osterhout) [inline docker-hub/deactivate-account.md:23]:
```suggestion
    1. Navigate to **Organizations** and then select the organization(s) you need to leave.
```

File: docker-hub/deactivate-account.md [hunk 4]
```diff
@@ -4,47 +4,65 @@ description: Learn how to deactivate a Docker Hub account or an organization
...
+    2. Find your username in the **Members** tab.
+    3. Select the **More options** menu and then select **Leave organization**.
 
-      In Docker Hub, navigate to **_Your Account_** > **Account Settings** > **Billing**.
+- If you are the sole owner of an organization,  either add someone to [the **owners** team](manage-a-team.md#the-owners-team) and then remove yourself from the organization, or deactivate the organization. Similarly, if you are the sole owner of a company, either add someone else as a company owner and then remove yourself, or deactivate the company.
```
Comment (craig-osterhout) [inline docker-hub/deactivate-account.md:27]:
```suggestion
- If you are the sole owner of an organization, either add someone to [the **owners** team](manage-a-team.md#the-owners-team) and then remove yourself from the organization, or deactivate the organization. Similarly, if you are the sole owner of a company, either add someone else as a company owner and then remove yourself, or deactivate the company.
```

File: docker-hub/image-management.md [hunk 5]
```diff
@@ -4,69 +4,34 @@ keywords: dashboard, images, image management, inactive
...
+Advanced Image Management allows you to manage Docker images across all repositories and streamline storage in Docker Hub.
 
-The Advanced Image Management dashboard provides a snapshot of your existing images and allows you to view, sort, and filter images by tags, activity status, and date, and contains options to clean up your workspace by deleting images that are no longer required.
+It provides:
+- A snapshot of your existing images
+- Allows you to view, sort, and filter images by tags, activity status, and date
+- Contains options to clean up your workspace by deleting images that are no longer required
 
-The dashboard provides visibility into the status of the images in your repository and allows you to reduce the number of images stored in Docker Hub by reviewing their active status. An image retains its 'active' status if it is pulled or pushed in the last month. If there isn’t any activity on the image in the last month, it is considered 'inactive'.
+## Access the Advanced Image Management dashboard:
```
Comment (craig-osterhout) [inline docker-hub/image-management.md:16]:
```suggestion
## Access the Advanced Image Management dashboard
```

File: docker-hub/image-management.md [hunk 6]
```diff
@@ -4,69 +4,34 @@ keywords: dashboard, images, image management, inactive
 title: Advanced Image Management dashboard
 ---
 
-{% include upgrade-cta.html
-  body="Docker Pro, Team, and Business users can now use Docker Hub's **Advanced Image Management** dashboard. The dashboard enables a new level of access to your content, allowing fine-grained pruning of old data and exploration of old versions of pushed images. Upgrade now to start using the Advanced Image Management dashboard."
-  header-text="This feature requires a paid Docker subscription"
-  target-url="https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade_image-mgmt"
-%}
+{% include upgrade-cta.html %}
```
Comment (craig-osterhout) [inline docker-hub/image-management.md:7]: Not sure if intentional. There is no more text in the box. Just the upgrade button.

File: docker-hub/deactivate-account.md [hunk 7]
```diff
@@ -4,47 +4,65 @@ description: Learn how to deactivate a Docker Hub account or an organization
...
 
-4. Unlink your [Github and Bitbucket accounts](../docker-hub/builds/link-source.md#unlink-a-github-user-account).
+Once you have completed all the steps above, you can deactivate your organization. 
 
-Once you have completed all the steps above, you may deactivate your organization. On Docker Hub, go to **Organizations** > **_Your Organization_** > **Settings** > **Deactivate Org**.
+1. On Docker Hub, select **Organizations**.
+2. Choose the organization you want to deactivate. 
+3. In**Settings**, select the **Deactivate Org** tab and then **Deactivate organization**.
 
 > This cannot be undone! Be sure you've gathered all the data you need from your organization before deactivating it.
```
Comment (craig-osterhout) [inline docker-hub/deactivate-account.md:67]: For consistency with the previous warning.
```suggestion
> This cannot be undone. Be sure you've gathered all the data you need from your organization before deactivating it.
```

File: docker-hub/deactivate-account.md [hunk 8]
```diff
@@ -4,47 +4,65 @@ description: Learn how to deactivate a Docker Hub account or an organization
...
-3. Remove all other members, including those in the **Owners** team, within the organization.
+### Deactivate
 
-4. Unlink your [Github and Bitbucket accounts](../docker-hub/builds/link-source.md#unlink-a-github-user-account).
+Once you have completed all the steps above, you can deactivate your organization. 
 
-Once you have completed all the steps above, you may deactivate your organization. On Docker Hub, go to **Organizations** > **_Your Organization_** > **Settings** > **Deactivate Org**.
+1. On Docker Hub, select **Organizations**.
+2. Choose the organization you want to deactivate. 
+3. In**Settings**, select the **Deactivate Org** tab and then **Deactivate organization**.
```
Comment (craig-osterhout) [inline docker-hub/deactivate-account.md:65]:
```suggestion
3. In **Settings**, select the **Deactivate Org** tab and then **Deactivate organization**.
```

File: docker-hub/deactivate-account.md [hunk 9]
```diff
@@ -4,47 +4,65 @@ description: Learn how to deactivate a Docker Hub account or an organization
...
+Once you have completed all the steps above, you can deactivate your account. 
+
+1. Select your account name in the top-right of Docker Hub and from the dropdown menu, select **Account Settings**.
+2. From the **Deactivate Account** tab, select **Deactivate account**. 
+
+> This cannot be undone. Be sure you've gathered all the data you need from your account before deactivating it.
 {: .warning }
 
 
 ## Deactivating an organization
```
Comment (craig-osterhout) [inline docker-hub/deactivate-account.md:46]: For consistency with 'Deactivate a user account'
```suggestion
## Deactivate an organization
```

File: docker-hub/convert-account.md [hunk 10]
```diff
@@ -4,60 +4,44 @@ title: Convert an account into an organization
...
 
-Before you convert a user account to an organization, ensure that you have completed the following steps:
+Before you convert a user account to an organization, ensure that you meet the following requirements:
 
-1. The user account that you wish to convert must not be a member of a company or any teams or organizations. You must remove the account from all teams, organizations, or the company.
+- The user account that you wish to convert must not be a member of a company or any teams or organizations. You must remove the account from all teams, organizations, or the company.
 
-    Go to **Organizations**, select an organization from the list, and then click the Leave organization arrow next to your username in the members list.
+    To do this:
+    1.  Navigate to **Organizations** and then select the organization(s) you need to leave.
```
Comment (craig-osterhout) [inline docker-hub/convert-account.md:25]:
```suggestion
    1. Navigate to **Organizations** and then select the organization(s) you need to leave.
```

File: docker-hub/image-management.md (hunk 6 above)
Comment (aevesdocker) [reply docker-hub/image-management.md:7]: Yes intentional as a lot of the body text just repeats what is said in the main body of the text and just made it look confusing

**Review by craig-osterhout — APPROVED** [review]
LGTM

**Pull request merged**